src/index.html

```diff
@@ -4,13 +4,40 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, 
+      maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
+    <title>Search bars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-container">
+      <form class="search-box" data-qa="big">
+        <label for="search-input" class="sr-only"></label>
+        <input 
+        type="text" 
+        id="search-input" 
+        class="search-input" 
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        >
+      </form>
+    </div>
+
+    <div class="search-container">
+      <form class="search-box-small" data-qa="small">
+        <label for="search-input-small" class="sr-only"></label>
+        <input 
+        type="text" 
+        id="search-input-small" 
+        class="search-input-small" 
+        placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </form>
+    </div>
   </body>
 </html>
+
+
```

Comment:
These empty lines are redundant. You need just one empty line at the end

Reply:
Not fixed 
<img width="715" alt="image" src="https://user-images.githubusercontent.com/99403198/230879581-268be00f-51b1-43d3-bcb9-1433e5836ca8.png">